public/app/features/alerting/unified/components/contact-points/ContactPoints.tsx

```diff
@@ -444,11 +443,17 @@ const ContactPointHeader = (props: ContactPointHeaderProps) => {
           </Text>
         </Stack>
         {isReferencedByAnyPolicy && (
-          <MetaText>
-            <Link to={createUrl('/alerting/routes', { contactPoint: name })}>
-              is used by <Strong>{numberOfPolicies}</Strong> {pluralize('notification policy', numberOfPolicies)}
-            </Link>
-          </MetaText>
+          <Text variant="bodySmall" color="secondary">
+            is used by{' '}
+            <TextLink
```

Comment:
`<TextLink>` doesn't do inheritance of weights and sizes because it defaults to the regular weight / size combo. Might be something we want to fix in the design system (with unknown consequences) so until then we have to repeat some props in the children 🤷